### PR TITLE
Use Rollup to generate UMD bundle

### PR DESCRIPTION
**Pull request by MattiasBuelens**
I wanted to use `stable` in a project that uses RequireJS, but unfortunately this package only supports CommonJS-style modules.

This PR solves this by adding a UMD wrapper around the main script file, so that it works in both CommonJS, AMD and web environments. I decided to use [Rollup](https://rollupjs.org/) because I like ES2015 `import` and `export`, but other solutions (e.g. Browserify) would also be possible.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - 0.10
+  - 4
```
Comment (MattiasBuelens) [inline .travis.yml]: Rollup requires Node.js 4 or higher, so I had to bump this up.

**Comment by stephank** [comment]
Gosh, sorry about completely missing this. I'll take a look at this soon!

**Comment by MattiasBuelens** [comment]
No worries, in the meantime I simply use my fork in my own projects. 😉 

I've added some additional commits to my fork outside of this PR. If you want, I can add them to this PR as well:
* Add a `module` entry point to `package.json` for using the library with an ESM loader and/or bundler (e.g. Rollup). (MattiasBuelens/stable@1044bb244a84e6064427fb8e9e78930cafc9fb87)
* Update the type definitions to use ESM `export default` instead of CommonJS `export =` (MattiasBuelens/stable@67210631127a3a5b99a5d3b5267329abf6bc03c4)

**Comment by MattiasBuelens** [comment]
There we go. Now, the library can be used as a CommonJS, AMD or ESM module. Oh, and of course as a global variable through a `<script>` tag. 😄

**Comment by stephank** [comment]
I just published 0.1.7. :)

**Pull request merged**